tutorials/ComplexArithmetic/Workbook_ComplexArithmetic.ipynb

```diff
@@ -0,0 +1,953 @@
+    "\n",
+    "**What you should know for this workbook**\n",
+    "\n",
+    "1. Basic math\n",
+    "2. Basic Python knowledge is usefull but not needed\n",
+    "\n",
+    "If you need to look up some formulas quickly, you can find them in [this cheatsheet](https://github.com/microsoft/QuantumKatas/blob/master/quickref/qsharp-quick-reference.pdf).\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook has several tasks that require you to write Python code to test your understanding of the concepts. If you are not familiar with Python, [here](https://docs.python.org/3/tutorial/index.html) is a good introductory tutorial for it. \n",
+    "\n",
+    "Let's start by importing some useful mathematical functions and constants, and setting up a few things necessary for testing the exercises. **Do not skip this step**.\n",
+    "\n",
+    "Click the cell with code below this block of text and press `Ctrl+Enter` (`⌘+Enter` on Mac)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Success!\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Run this cell using Ctrl+Enter (⌘+Enter on Mac).\n",
+    "from testing import exercise\n",
+    "from typing import Tuple\n",
+    "\n",
+    "import math\n",
+    "\n",
+    "Complex = Tuple[float, float]\n",
+    "Polar = Tuple[float, float]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Algebraic Perspective\n",
+    "\n",
+    "## Imaginary numbers\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### <span style=\"color:blue\">Exercise 1</span>: Powers of $i$.\n",
+    "\n",
+    "**Input:** An even integer $n$.\n",
+    "\n",
+    "**Goal:** Return the $n$th power of $i$, or $i^n$.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Solution \n",
+    "\n",
+    "When raising $i$ to a higher power then 2, the answer will not always be $-1$. For example, when raising $i$ to the power of 4 gives: $i^4 = i^2 \\cdot i^2$ which would give: $ -1 \\cdot -1 = 1$. \n",
+    "\n",
+    "A generalized rule could be made: when the power to which you raise $i$ is fully dividable by 4 the result will always be 1. This is the complete pattern that arises when taking $i$ to higher powers:\n",
+    "\n",
+    "|Power of I|$i^1$|$i^2$|$i^3$|$i^4$|$i^5$|$i^6$|$i^7$|$i^8$|$\\dots$|\n",
+    "|----|----|----|----|----|----|----|----|----|----|\n",
+    "|Result|$i$|$-1$|$i$|$1$|$i$|$-1$|$i$|$1$|$\\dots$|\n",
+    "\n",
+    "This means that $i^{(n \\cdot 4)}$ will always give $1$ if $n$ is a whole number.\n",
+    "\n",
+    "*Python note*:  \n",
```

Comment:
I think this sentence would look nicer if it was the first line of the note (same comment goes for other Python notes)